managed/ui/src/components/tables/RestoreBackup/RestoreBackup.js

```diff
@@ -60,6 +61,15 @@ export default class RestoreBackup extends Component {
     return isNonEmptyObject(this.props.backupInfo);
   };
 
+  /**
+   * This is an onchange event for storage type.
+   * 
+   * @param {string} value Input field value.
+   */
+   backupConfigType = (value) => {
+    this.props.initialValues.storageConfigUUID = value;
+  }
```

Comment:
Why are we modifying the initialValues to hold current state? Just let Formik handle the state of the form.

Reply:
This is a change event. I don't think Formik will handle that. I tried removing it but didn't work as expected.

Reply:
Try using `YBFormSelect` instead of `YBSelectWithLabel`